geometa/rest.py

```diff
@@ -58,12 +60,30 @@ def get_documents_by_radius(user, latitude, longitude, radius):
     return _find(user, query)
 
 
-def _get_girder_path(girder_file):
+def _get_path_from_filesystem(girder_file):
```

Comment:
There is a Girder File model method `File().getLocalFilePath(file)` that can be used instead of querying the assetstore adapter.  This has the virtue that if `girder mount` is set up to provide a FUSE, the files are read through that (which can involve less data transfer than downloading).

Reply:
@manthey I changed it to use the getLocalFilePath method. Does this look good to you?